### PR TITLE
[Feature] More API tests and some cleanup

### PR DESCRIPTION
**Pull request by aaxelb**

### CODE REVIEW

File: reviews/admin.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.contrib import admin
+# from django.contrib import admin
```
Comment (sheriefvt) [inline reviews/admin.py]: can you remove instead of commenting

File: docker-compose.yml [hunk 2]
```diff
@@ -251,7 +251,7 @@ services:
 
   reviews:
     image: quay.io/centerforopenscience/ember-osf-reviews:develop-development-local
-    command: /bin/bash -c "yarn --pure-lockfile --no-progress --no-emoji && ./node_modules/bower/bin/bower install --allow-root --config.interactive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4400"
```
Comment (laurenbarker) [inline docker-compose.yml:254]: Woo!

File: reviews/admin.py (hunk 1 above)
Comment (laurenbarker) [inline reviews/admin.py]: Why is this commented out?

Comment (aaxelb) [reply reviews/admin.py]: Yeah, I'll just delete the file

Comment (aaxelb) [reply reviews/admin.py]: Because flake8 complained, and I hesitated to delete the file.

File: reviews/serializers.py [hunk 3]
```diff
@@ -20,23 +19,43 @@
 
 
 # Pseudo-class to hide the creator field if it shouldn't be shown
-def IfCanViewLogCreator(field_cls):
-    class IfCanViewLogCreatorField(field_cls):
+def UnlessCommentsAnonymous(field_cls):
```
Comment (laurenbarker) [inline reviews/serializers.py]: Is "Unless" fields used other places? It seems a little strange.

File: reviews/admin.py (hunk 1 above)
Comment (laurenbarker) [reply reviews/admin.py]: Oh I get it, it's the only thing in the file.

File: reviews/serializers.py (hunk 3 above)
Comment (aaxelb) [reply reviews/serializers.py]: It's not, but "IfCanView" bothered me. Looks like there are some `HideIf...` and `ShowIf...` fields elsewhere; do `HideIfCommentsAnonymous` and `HideIfCommentsPrivate` seem better?

Comment (laurenbarker) [reply reviews/serializers.py]: That does seem a little better; less logic to think through. Up to you though, I don't feel strongly either way.

File: docker-compose.yml [hunk 4]
```diff
@@ -251,7 +251,7 @@ services:
...
+    command: /bin/bash -c "yarn --pure-lockfile --no-progress --no-emoji && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4400"
```
Comment (chrisseto) [inline docker-compose.yml:254]: Worth commenting but probably not worth changing. `yarn run ember server ...` will work as well.

**Pull request merged**